manifests/agent.pp

```diff
@@ -54,11 +54,12 @@
         source  => 'puppet:///modules/check_mk/local_checks',
     }
 
-    # Copy systemd plugin
-    file { '${plugin_active_location}/mk_systemd_status':
-        ensure  => file,
+    # copy plugins directory
+    file { "${plugin_active_location}":
```

Comment:
I was worried about this because above in the file, it specified another file to be put in the plugins directory. As long as these 2 rules do not compete, then I'm happy with this change.